scio-avro/src/main/scala/com/spotify/scio/avro/AvroTypedIO.scala

```diff
@@ -21,6 +21,7 @@ import com.spotify.scio.avro.types.AvroType.HasAvroAnnotation
 import com.spotify.scio.coders.Coder
 import com.spotify.scio.io.{ScioIO, Tap, TapOf, TapT}
 import com.spotify.scio.values.SCollection
+import magnolify.avro.{AvroType => MagnolifyAvroType}
```

Comment:
```suggestion
import magnolify.avro.{AvroType => AvroMagnolifyType}
```
would fit better with proposed IO naming